docs/gettinginvolved/cluster-setup.rst

```diff
@@ -199,6 +171,79 @@ Or you can run, for example, the acceptance tests against the created cluster:
      --flocker-version='' \
      flocker.acceptance.obsolete.test_containers.ContainerAPITests.test_create_container_with_ports
 
+==============================
```

Comment:
This is a sub-section, so doesn't require the ===== above the title (just below it)
We also use title case on our headings, so the title should be "Adding Containers and Datasets"